src/dashboardUsers/path/dashUsers-routes.js

```diff
@@ -25,13 +26,13 @@ router
 // View All Users by only Admin
 router
   .route("/view-users")
-  .get(authorizeUser(process.env.ADMIN_ROLE),listAllUsers(), (req, res) => {
+  .get(authorizeUser(process.env.ADMIN_ROLE),paginateUser(userModel), (req, res) => {
     res.status(200).json({
       error: null,
       data:[{
```

Comment:
I don't see the point here for `data` to be a list containing one object that contains a list of users. Better for `data` to just be the object.